.travis.yml

```diff
@@ -5,7 +5,7 @@ matrix:
     - compiler: clang
       env:
         - CXX_COMPILER='clang++-3.9'
-        - PYTHON_VER='2.7'
+        - PYTHON_VER='3.5'
```

Comment:
I see the following travis builds:
```
# Allowed failure
        - CXX_COMPILER='clang++-3.9'
        - PYTHON_VER='3.5'

#1
      - CXX_COMPILER='clang++-3.6'
      - PYTHON_VER='3.5'

#2
      - CXX_COMPILER='clang++-3.6'
      - PYTHON_VER='3.5'

#3
      - CXX_COMPILER='clang++-3.9'
      - PYTHON_VER='3.5'

#4
      - CXX_COMPILER='g++-4.9'
      - PYTHON_VER='2.7'

#5
      - CXX_COMPILER='g++-6'
      - PYTHON_VER='3.5'
```

Unless I am missing something #1 and #2 are duplicates and #3 shouldn't work?